tools/doxygen/doxygen.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+LAMMPSBASE=${PWD} ;
+LAMMPSSRC="${LAMMPSBASE}/src" ;
+LAMMPSDOC="${LAMMPSBASE}/doc" ;
+LAMMPSDOCHTML="${LAMMPSDOC}/html" ;
+LAMMPSDEVELOPERDOC="${LAMMPSBASE}/doc/src/Developer" ;
+LAMMPSDOCTMP="/tmp/lammps-docs-*" ;
+LAMMPSDOCMAKE="${LAMMPSDOC}/Makefile" ;
+LAMMPSDOXYGEN="${LAMMPSBASE}/tools/doxygen" ;
+LAMMPSDOXYGENDOC="${LAMMPSDOXYGEN}/doc" ;
+LAMMPSDOXYGENDOCMANUAL="${LAMMPSDOXYGEN}/doc/html/Manual" ;
+
+LAMMPSDOXYFILE=${LAMMPSDOXYGEN}/Doxyfile.lammps ;
+DOXYFILE=${LAMMPSDOXYGEN}/Doxyfile ;
+
+LAMMPSDEVELOPERDOXFILE=${LAMMPSDOXYGEN}/Developer.dox.lammps ;
+DEVELOPERDOXFILE=${LAMMPSDOXYGEN}/Developer.dox ;
+
+FIG2DEV=`which fig2dev` ;
+DOXYGEN=`which doxygen` ;
```

Comment:
In bash you can use `type -p` instead of `which`.

Reply:
`which ...` works also fine.